libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverviewChartPivot.tsx

```diff
@@ -178,6 +179,17 @@ export class ModelOverviewChartPivot extends React.Component<
             selectedMetric={this.state.selectedMetric}
           />
         </PivotItem>
+        {(this.context.modelMetadata.modelType === ModelTypes.Binary ||
```

Comment:
Can we disable the confusion matrix logic for large data with the function `IfEnableLargeData()`? We don't support confusion matrix in SDK backend as yet.

Reply:
Sure. Can you create a work item to track it @gaugup ?

Reply:
Added the extra check